File: Janari0.Services/Database/Output.cs
using System;
using System.Collections.Generic;

namespace Janari0.Services.Database;

public partial class Output
{
    public int OutputId { get; set; }

    public DateTime? Date { get; set; }

    public string? PaymentMethod { get; set; }

    public bool? Concluded { get; set; }

    public float? Amount { get; set; }

    public string? ReceiptNumber { get; set; }

    public int? BuyerId { get; set; }

    public int? OrderId { get; set; }

    public virtual Buyer? Buyer { get; set; }

    public virtual Order? Order { get; set; }

    public virtual ICollection<OutputItem> OutputItems { get; } = new List<OutputItem>();
}

File: janari0_mobile/lib/model/output.dart
import 'package:json_annotation/json_annotation.dart';

part 'output.g.dart';

@JsonSerializable()
class Output {
  int? outputId;
  DateTime? date;
  String? paymentMethod;
  bool? concluded;
  double? amount;
  String? receiptNumber;
  int? buyerId;
  int? orderId;

  Output();

  factory Output.fromJson(Map<String, dynamic> json) => _$OutputFromJson(json);

  /// Connect the generated [_$OutputToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$OutputToJson(this);
}
